packages/ui/src/components/app-sidebar/sidebar-item.tsx

```diff
@@ -94,15 +93,12 @@ export const SidebarItem = ({
       {!collapsed && (
         <DropdownMenu.Root>
           <DropdownMenu.Trigger asChild>
-            <Sidebar.MenuAction
-              className={cn('text-sidebar-icon-3 hover:text-sidebar-icon-1', isInset ? 'right-[3px]' : 'right-0')}
-              showOnHover
-            >
+            <Sidebar.MenuAction className={cn('text-sidebar-icon-3 hover:text-sidebar-icon-1 right-[3px]')} showOnHover>
```

Comment:
Looks like we don't need `cn` here anymore.
```suggestion
            <Sidebar.MenuAction className="text-sidebar-icon-3 hover:text-sidebar-icon-1 right-[3px]" showOnHover>
```

Reply:
Fixed